OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -59,10 +59,9 @@ public abstract class SupportPowerInfo : ConditionalTraitInfo
 		public readonly string BeaconImage = "beacon";
 		[SequenceReference("BeaconImage")] public readonly string BeaconPoster = null;
 		[PaletteReference] public readonly string BeaconPosterPalette = "chrome";
-		[SequenceReference("BeaconImage")] public readonly string ClockSequence = "clock";
-
-		[SequenceReference("BeaconImage")] public readonly string ArrowSequence = "arrow";
-		[SequenceReference("BeaconImage")] public readonly string CircleSequence = "circles";
+		[SequenceReference("BeaconImage")] public readonly string ClockSequence = null;
+		[SequenceReference("BeaconImage")] public readonly string ArrowSequence = null;
+		[SequenceReference("BeaconImage")] public readonly string CircleSequence = null;
```

Comment:
I know upgrade rules are a pain, but in my opinion changing the defaults needs at least a console warning message, changing defaults silently might catch some modders off guard.